kaldi/nnet3/decodable-online-looped.clif

```diff
@@ -48,3 +56,12 @@ from "nnet3/decodable-online-looped.h":
 
       def `FrameSubsamplingFactor` as frame_subsampling_factor(self) -> int:
         """Returns the frame subsampling factor."""
+
+      def `LogLikelihoods` as log_likelihoods_frame(self, frame: int) -> Vector:
+        """Returns the log-likelihoods for the given frame"""
+	return _vector_wrapper(...)
+
+      def `LogLikelihoods` as log_likelihoods(self, frame_from: int, frame_to: int) -> Matrix:
```

Comment:
It would be better if this method were to accept frame-offset and num-frames as arguments instead of frame-from/frame-to.

Reply:
Why is that, is that more kaldi-like?